pynamodb/tests/test_model.py

```diff
@@ -515,7 +524,7 @@ def test_delete(self):
                 'TableName': 'UserModel'
             }
             args = req.call_args[0][1]
-            self.assertEqual(args, params)
+            deep_eq(args, params, _assert=True)
```

Comment:
We can, it was someone else's gist, so I tried to modify it as little as possible.  (I had to use six for python3 compatibility)